docs/operator/configuration.md

```diff
@@ -0,0 +1,68 @@
+# Configuration
+
+Learn about the default configuration in Busola and [how to change it](#changing-the-configuration).
+
+## Configuration Sources
+
+Busola configuration is the product of gathering and merging the configurations from several individual sources. The following list presents the sources in the order of precedence:
+
+### Backend
+
+- Busola backend default cluster configuration, acquired from the [defaultConfig.yaml](../../backend/settings/defaultConfig.yaml) file.
+- Busola cluster configuration, available on the Busola cluster in the ConfigMap "busola/busola-config" under the key "config".
```

Comment:
Aren't busola/busola-config and config parameters or sections in code files? I wouldn't use inverted commas with them.